plugins/meta/tuning/tuning_test.go

```diff
@@ -1171,5 +1171,51 @@ var _ = Describe("tuning plugin", func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 
+		It(fmt.Sprintf("[%s] does not allow ifname with path separator", ver), func() {
+			conf := []byte(fmt.Sprintf(`{
+				"name": "test",
+				"type": "tuning",
+				"cniVersion": "%s",
+				"sysctl": {
+					"net.ipv4.conf.all.log_martians": "1"
+				},
+				"prevResult": {
+					"interfaces": [
+						{"name": "dummy0", "sandbox":"netns"}
```

Comment:
You may need to change this name 'dummy0' to "eth/0" as well as "args.IfName" (because usual CNI chained execution, prevResult interface name should be same as args.IfName).

Reply:
Changed.
In case of this test it does not matter, but better to keep it the correct way.
Thanks